packages/eslint-plugin/package.json

```diff
@@ -63,23 +61,22 @@
         "@workleap/typescript-configs": "workspace:*",
         "eslint": "8.57.0",
         "jest": "29.7.0",
-        "ts-node": "10.9.2",
         "tsup": "8.0.2"
     },
     "dependencies": {
-        "@typescript-eslint/eslint-plugin": "^7.6.0",
-        "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-jest": "^28.2.0",
+        "@eslint/js": "^9.0.0",
+        "eslint-flat-config-utils": "^0.2.4",
+        "eslint-plugin-jest": "^28.3.0",
         "eslint-plugin-jsx-a11y": "^6.8.0",
         "eslint-plugin-mdx": "^3.1.5",
         "eslint-plugin-package-json": "^0.12.2",
         "eslint-plugin-react": "^7.34.1",
-        "eslint-plugin-react-hooks": "^4.6.0",
+        "eslint-plugin-react-hooks": "^4.6.2",
         "eslint-plugin-storybook": "^0.8.0",
-        "eslint-plugin-testing-library": "^6.2.1",
-        "eslint-plugin-yml": "^1.14.0",
+        "eslint-plugin-testing-library": "^6.2.2",
+        "globals": "^15.1.0",
         "jsonc-eslint-parser": "^2.4.0",
-        "yaml-eslint-parser": "^1.2.2"
+        "typescript-eslint": "^7.8.0"
```

Comment:
i can't remember. The inspiration project i looked at was shopify's and they have it in dependencies, not peer dependencies. Most likely we changed in because we had issues with versionning in our projects